schemas/BaseElements.xsd

```diff
@@ -1272,6 +1272,19 @@
 												<xs:element minOccurs="0" name="UsedForWholeBuildingVentilation" type="HPXMLBoolean"/>
 												<xs:element minOccurs="0" name="UsedForSeasonalCoolingLoadReduction" type="HPXMLBoolean"/>
 												<xs:element minOccurs="0" name="UsedForGarageVentilation" type="HPXMLBoolean"/>
+												<xs:element minOccurs="0" name="IsSharedMultifamilySystem" type="HPXMLBoolean"/>
```

Comment:
I suggest just calling this `IsSharedSystem` (since it can apply to single-family attached as well), with a note that a shared system is defined as serving multiple building/dwelling units.

Reply:
Here's what I have in another branch: https://github.com/hpxmlwg/hpxml/pull/233/files#diff-45b652e6a591f5d25c2331068b810566R3045-R3049